lib/core/datatable/components/datatable/datatable.component.html

```diff
@@ -1,25 +1,26 @@
 <div
     role="grid"
     *ngIf="data" class="adf-full-width"
-    [class.adf-data-table-card]="display === 'gallery'"
-    [class.adf-data-table]="display === 'list'"
-    [class.adf-data-table--empty]="!isHeaderVisible()">
+    [class.adf-datatable-card]="display === 'gallery'"
+    [class.adf-datatable-list]="display === 'list'"
+    [class.adf-sticky-header]="stickyHeader"
+    [class.adf-datatable--empty]="!isHeaderVisible()">
     <div *ngIf="showHeader && isHeaderVisible()" class="adf-datatable-header" role="rowgroup">
-        <div class="adf-datatable-row" *ngIf="display === 'list'" role="row">
+        <div class="adf-datatable-row" *ngIf="display === 'list'" role="adf-datatable">
```

Comment:
there's no such role in the Accessibility as "adf-datatable" ;) use "row" as it is the standard identifier for screen readers